surf_tide/src/main.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2020 O.S. Systems Sofware LTDA
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use derive_more::From;
+use futures_util::lock::Mutex;
+use http_client::native::NativeClient;
+use std::sync::Arc;
+
+use rwcst::prelude::*;
+
+#[tokio::main]
+async fn main() {
+    let (url, _guards) = rwcst::start_remote_mock();
+    let local_client = LocalClient::new();
+    let remote_client = RemoteClient::new(&url);
+    let app = App::new(remote_client);
+    rwcst::run(local_client, app).await;
```

Comment:
The execution seems to be locked, running `cargo run -p surf_tide` from workspace's root doesn't finish. I suggest looking into the spawned future for the tide server and also checking the routes externally to se if it's responding.